### DISCUSSION TITLE
Fans Run at Full RPM after Startup

### DISCUSSION THREAD

**Discussion by tsmit008** [Rem0o/FanControl.Releases#1907] [Q&A]
Hi all, 

When I boot to windows after the PC has been disconnected from AC power (ie when I switch the PSU off) the fans run at full RPM despite Fan Control trying to slow them down. I have them set to "force apply" but this does nothing. Everything works fine when I restart the computer, or if the computer has been turned on without being cut off from AC power.  Does anyone know what causes this? Thanks.

**Comment by Rem0o** [comment]
I wouldn't think getting power cut is considered normal operation for your motherboard.  It's unexpected behavior at that point.  Might leave it in a limbo state.
